docs/types.rst

```diff
@@ -120,7 +120,8 @@ Exponentiation
 Exponentiation is only available for unsigned types. Please take care that the types
 you are using are large enough to hold the result and prepare for potential wrapping behaviour.
 
-Note that ``0**0`` is defined by the EVM as ``1``.
+.. note::
+  Note that ``0**0`` is defined by the EVM as ``1``.
```

Comment:
Let me now if this doesn't make sense. I think this stands out and deserves its highlighting block.

Reply:
I think it makes sense